### PR TITLE
add GateChain mainnet&testnet

### PR DESCRIPTION
**Pull request by vincentbdb**
add the chain info of GateChain mainnet & testnet
tks

### CODE REVIEW

File: _data/chains/eip155-86.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    ],
+    "explorers": [
+      {
+        "name": "GateScan",
+        "url": "https://www.gatescan.org",
+        "standard": "EIP3091"
+      }
+    ],
+    "infoURL": "https://www.gatechain.io"
+  },
```
Comment (ligi) [inline _data/chains/eip155-86.json:27]:
```suggestion
  }
```

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-85.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    ],
+    "explorers": [
+      {
+        "name": "GateScan",
+        "url": "https://www.gatescan.org/testnet",
+        "standard": "EIP3091"
+      }
+    ],
+    "infoURL": "https://www.gatechain.io"
+  },
```
[the remark this answers is not in the document]
Comment (vincentbdb) [reply _data/chains/eip155-85.json]: tks

File: _data/chains/eip155-86.json (hunk 1 above)
Comment (vincentbdb) [reply _data/chains/eip155-86.json]: tks

**Review by ligi — CHANGES_REQUESTED** [review]
the shortname must be unique

**Review by ligi — APPROVED** [review]

**Pull request merged**